app/routes.py

```diff
@@ -1,2 +1,250 @@
-from flask import Blueprint
+from app import db
+from flask import Blueprint, request, make_response, jsonify
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
 
+
+
+
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rental_bp=Blueprint("rentals",__name__, url_prefix="/rentals")
+
+
+
+@customer_bp.route("", methods=["POST"])
+def valid_customer():
+    form_data = request.get_json()
+    if "name" not in form_data\
+        or "postal_code" not in form_data\
+            or "phone" not in form_data:
+        return make_response({"details":"Invalid data"}, 400)
+    else:
+        new_customer = Customer(name=form_data["name"],
+                                postal_code=form_data["postal_code"],
+                                phone=form_data["phone"])
+    db.session.add(new_customer)
+    db.session.commit()
+    return make_response({"id": new_customer.customer_id}, 201)
```

Comment:
The conditional logic here is a little off.  Having lines 27-29 outside of the if and having the first conditional return will work, but it doesn't quite flow.  I recommend either include lines 27-29 in the else, or remove the else and have line 24 at the same tab level as 27-29.